project/Dependencies.scala

```diff
@@ -124,17 +121,18 @@ object Dependencies {
     * Related Slick PR: https://github.com/slick/slick/pull/2101
   */
   private val slickV = "3.3.2-2076hotfix" // scala-steward:off (CROM-6620)
-  private val snakeyamlV = "1.28"
-  private val specs2MockV = "4.10.6"
+  private val snakeyamlV = "1.29"
+  private val specs2MockV = "4.12.3"
```

Comment:
I care about this, but I cannot find any reliable documentation links about specs2-mock or any version history. I hope they didn't change anything that will cause any of our tests to pass when they should fail.

Reply:
I care too. The reason it's hard to find docs on `specs2-mock` is that it seems to be [End Of Life](https://github.com/etorreborre/specs2/commits/master/mock/). See the cromwell sibling PR that kills this dependency off 😉🔪💀

As for some sort of docs:
- 4.12.0 mentions specs2's `specs2-mock` extension: https://etorreborre.github.io/specs2/guide/SPECS2-4.12.0/org.specs2.guide.Installation.html#other-dependencies
- 5.0 (in RC) no longer mentions the extensions and instead refers to `scalamock`: https://etorreborre.github.io/specs2/guide/5.0.0-RC-01/org.specs2.guide.Installation.html#other-dependencies